src/GovUk.Education.ExploreEducationStatistics.Admin/Controllers/Api/ReleaseVersionsController.cs

```diff
@@ -179,14 +202,14 @@ public async Task<ActionResult<List<DataSetUploadResultViewModel>>> UploadDataSe
                 .HandleFailuresOrOk();
         }
 
-        [HttpPost("releases/{releaseVersionId:guid}/import-data-sets")]
-        public async Task<ActionResult<List<DataFileInfo>>> ImportBulkZipDataSetsFromTempStorage(
+        [HttpPost("releaseVersions/{releaseVersionId:guid}/import-data-sets")]
+        public async Task<ActionResult<Unit>> ImportDataSetsFromTempStorage(
```

Comment:
As per the DELETE comment, if wanting to return no-body POST responses, probably just return a non-generics ActionResult and use `HandleFailuresOrNoContent()`, but with a slight tweak:

```
HandleFailuresOrNoContent(convertNotFoundToNoContent: false)
```